compiler/daml-lf-verify/src/DA/Daml/LF/Verify/Subst.hs

```diff
@@ -0,0 +1,157 @@
+-- Copyright (c) 2020 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+-- SPDX-License-Identifier: Apache-2.0
+
+-- | Term substitions for DAML LF static verification
+module DA.Daml.LF.Verify.Subst
```

Comment:
We're about to land another implementation of substitution for DAML-LF in https://github.com/digital-asset/daml/pull/5759. We should switch the code here over to use @sofiafaro-da's implementation once it has landed.